src/utils/hooks/useGetQuestionReviewHistory.tsx

```diff
@@ -39,6 +39,13 @@ const useGetQuestionReviewHistory = ({ isApplicant, ...variables }: UseGetQuesti
     ? useGetHistoryForApplicantQuery({ variables })
     : useGetHistoryForReviewerQuery({ variables })
 
+    const localisedResponse: { [key: string]: string} = {
+      'APPROVE': strings.LABEL_REVIEW_APPROVED,
+      'DECLINE': strings.LABEL_REVIEW_DECLINED,
+      'AGREE': strings.LABEL_CONSOLIDATION_AGREEMENT,
+      'DISAGREE': strings.LABEL_CONSOLIDATION_DISAGREEMENT
+    }
```

Comment:
Sure! I didn't find where it was! Thank you. Maybe I should add this to the new Localisations docs, so it helps for future work done around that : )

Reply:
That localisation doc is currently written from a user perspective, so I don't think we should clutter it up with internal details. Feel free to start an internal "localisation system" doc, although it'd need to have an explanation of how the localisation context works, how plugins get their strings, etc. so not sure you want to do that just yet.

I guess what I'm saying is it'd be weird to document this one very specific thing when there's more fundamental aspects of the localisation system we haven't yet documented.

Reply:
I have added to the same doc, ideally this is just basic material for dev/admins to use. A User Guide to be done separately for final users - where some of this work can be copied/based from. 

I didn't go into much details of contexts. I think we can have special "Learning" sections on the App to talk about technologies in use, instead of adding as part some specific note about how to add/do management of localisations